hubspot/plugin.py

```diff
@@ -454,57 +454,95 @@ class HubSpotSearchCondition:
         "LTE",
         "GT",
         "GTE",
+        "HAS_PROPERTY",
+        "NOT_HAS_PROPERTY",
     ]
     value: HubSpotPropertyValue
 
 
-@purpose("Search contacts.")
+@dataclass
+class AndGroup:
+    """A group of conditions that must ALL be true (AND logic). Maximum of 6 conditions"""
```

Comment:
Should we validate the 6 conditions constraint?

Reply:
I've added additional validation.

The API will also raise an error with very similar error messages.